cli/src/tests/corpus_test.rs

```diff
@@ -115,6 +115,12 @@ pub fn test_language_corpus(
     skipped: Option<&[&str]>,
     language_dir: Option<&str>,
 ) {
+    if let Some(filter) = LANGUAGE_FILTER.as_ref() {
+        if language_name != filter.as_str() {
```

Comment:
Don't think you *need* the `as_str` call

```suggestion
        if language_name != filter {
```

Reply:
Thanks for the comment. Also fixed the part I was copying from.